test/compilable/vcg-ast.d

```diff
@@ -1,6 +1,7 @@
 module vcg;
 // REQUIRED_ARGS: -vcg-ast -o-
 // PERMUTE_ARGS:
+// POST_SCRIPT: rm -f compilable/vcg-ast.d.cg && cat
```

Comment:
Looks like the temporary filename is always appended here :/

Reply:
I though the `-f` was supposed to handle that.  Is `rm` aliased to something else on the auto-tester?

Reply:
I think you'll have to write a `.sh` file for this and put it in `/extra-files/` to make it work on all platforms.